app/models/inviter.rb

```diff
@@ -0,0 +1,15 @@
+class Inviter
+  def initialize(creator, organization, team_id, team_title)
+    @creator      = creator
+    @organization = organization
+    @team_id      = team_id
+    @team_title   = team_title
+  end
+
+  def create_invitation
+    github_team = GitHubTeam.new(@creator.github_client, @organization.github_id)
+    team        = github_team.find_or_create_team(@team_id, @team_title)
```

Comment:
If you used a class method, this would be `team = GitHubTeam.find_or_create(creator.github_client, organization.github_id, team_id, team_title)`